faunadb/client.go

```diff
@@ -163,6 +163,9 @@ func NewFaunaClient(secret string, configs ...ClientConfig) *FaunaClient {
 
 	if client.queryTimeoutMs > 0 {
 		client.headers["X-Query-Timeout"] = strconv.FormatUint(client.queryTimeoutMs, 10)
+	} else {
+		client.queryTimeoutMs = uint64(requestTimeout.Milliseconds())
+		client.headers["X-Query-Timeout"] = strconv.FormatUint(uint64(requestTimeout.Milliseconds()), 10)
```

Comment:
That line is duplicated, it’s called in both branches of the if, why don’t you move it outside if and just update `client.queryTimeoutMs`

Reply:
I believe my comment on request vs query timeout also applies here: https://github.com/fauna/faunadb-jvm/pull/298#discussion_r583630192

Reply:
@erickpintor @marrony I updated PR, pls review.